### ISSUE TITLE
how to build tutorial

### ISSUE THREAD

**Issue by ramcharan0** [aliemranjazib/women-safety-app-sos-flutter#1]
greate work sir . i really appericate it . but can u brefily explain how to build https://github.com/HassamTalha/Amaan-Women-Safety-App this project. like what version of sdk did u used and can u brefily explain to me . thank u in advance

**Comment by aliemranjazib-zz** [comment]
our women safety app is enhanced version of amaan-women-app. and flutter
version is very latest i am usiing

**Comment by ramcharan0** [comment]
yeah i just saw it in ur youtube channel . but i want to use the amaan-women-app as a base and build  on top of it.
i really appericiate it if u can help me build the apk using that repo.
thank u sir and btw i just sub to your yt channel

**Comment by aliemranjazib-zz** [comment]
Thanks a lot.
my base project is also amaan safety

**Comment by ramcharan0** [comment]
so i want to build it on the lastest flutter sdk , can u just explain berifly pls..

**Comment by hanishx11** [comment]
hi sir, I'm facing issues while running the source code of the women safety app can you please help me to run this application.
